### PR TITLE
:sparkles: Set creator as author of report

### PR DESCRIPTION
**Pull request by ChristianJoe**

### CODE REVIEW

**Comment by LizzAy** [comment]
Shouldn't this field be automatically filled when the report is saved? Now the author must be filled from the user because it's mandatory...
Additionally when testing I get error "table cms_report has no column named author" or "no such column: cms_report.author". This should be fixed.

File: app/cms/migrations/0030_report_author.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+    dependencies = [
+        ('cms', '0029_auto_20180719_1248'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='report',
+            name='author',
+            field=models.CharField(default='Susanne Schwarzbach', max_length=200, verbose_name='Autor'),
```
Comment (mraerino) [inline app/cms/migrations/0030_report_author.py]: Der Default-Wert kommt mir komisch vor

File: app/cms/models/report.py [hunk 2]
```diff
@@ -43,6 +43,8 @@ class Meta:
     delivered = models.BooleanField(
         'Versendet', null=False, default=False)
 
+    author = models.CharField('Autor', max_length=200, null=False)
```
Comment (mraerino) [inline app/cms/models/report.py]: Bin ja für `Autor*in`

Comment (ChristianJoe) [reply app/cms/models/report.py]: Das Feld wird nicht im ADMIN angezeigt. Lässt sich aber einrichten.

**Pull request merged**